components/camel-jms/src/test/java/org/apache/camel/component/jms/JmsMultipleConsumersTest.java

```diff
@@ -42,7 +45,7 @@ public void configure() {
         context.start();
 
         // give it a bit time to setup both topic listeners
-        Thread.sleep(2000);
+        Awaitility.await().atMost(2, TimeUnit.SECONDS).until(context::isStarted);
```

Comment:
After `context.start()`, `context.isStarted()` should return `true`, right? If so I guess it is a no op

Reply:
Yeah correct - maybe the sleep can be removed all together

Reply:
Thanks, I'll give it a try tomorrow without the loop.